package.json

```diff
@@ -49,10 +50,12 @@
     "striptags": "3.1.1"
   },
   "devDependencies": {
+    "@babel/cli": "7.0.0-beta.44",
+    "@babel/core": "7.0.0-beta.44",
     "@percy-io/percy-storybook": "1.3.5",
     "@storybook/addon-viewport": "^3.4.1",
     "@storybook/addons": "3.4.4",
-    "babel-cli": "6.26.0",
+    "babel-core": "7.0.0-bridge.0",
```

Comment:
is `babel-core` and `@babel/core` the same?

Reply:
`babel-core@7.0.0-bridge.0` is a bridge from the old babel-core to the new. Is needed for e.g. storybook.

Reply:
ok, si it's correct to use both @babel/core and babel-core?

Reply:
Yes, storybook won't run without both. https://www.npmjs.com/package/babel-core/v/7.0.0-bridge.0